index.js

```diff
@@ -8,7 +8,20 @@ Object.assign(exports, {
   ],
   loadPlugin: function() {
     exports['migrator:migration:hook:require'] = function() {
-      require('babel-register');
+
+      try {
+        require('@babel/register');
+      }
+      catch (error) {
+
+        try {
+          require('babel-register');
+        }
+        catch {
+          throw new Error("You must include either babel-register or @babel/register in your "
+            + "package.json file.");
```

Comment:
Good, but it should probably just say you need to install them rather than specify adding them to package.json?

Reply:
Good call.

Reply:
Getting that on one line makes me happier anyway.